src/core/sagas/initializeSagas.js

```diff
@@ -0,0 +1,31 @@
+
+import { takeEvery, put } from 'redux-saga/effects';
+import  { get } from "lodash";
+
+import { domainName } from "../token";
+import { INITIALIZE_ACTION, initializeAction } from "../actions/initializeAction";
+
+export default takeEvery(INITIALIZE_ACTION.REQUEST, function*(action) {
+    const url = domainName + '/api/initialise';
+    let options = {};
+    options.method = "GET";
+    if (!options.headers) {
+        options.headers = new Headers({ Accept: 'application/json' });
+    }
+    options.headers = {
+        'X-Requested-With': "XMLHttpRequest",
```

Comment:
Strange. Does the API require that you pass this header?

Reply:
Yes, API requires it. I checked - it doesn't work without this header.